UI/data/themes/Acri.qss

```diff
@@ -269,6 +269,11 @@ OBSBasicSettings #PropertiesContainer {
     background-color: palette(dark);
 }
 
+/* EdiableList inside Group Fix, issues#8302 */
```

Comment:
This sort of comment is not needed in qss files, especially as this is a flaw of the theme and not a bug per se

Reply:
> This sort of comment is not needed in qss files, especially as this is a flaw of the theme and not a bug per se

thx, I will pay attention next time